src/components/pages/data/state-data.js

```diff
@@ -23,21 +23,21 @@ const State = ({ state }) => (
     <UnstyledList>
       {state.twitter && (
         <li>
-          <a href={`https://twitter.com/${state.twitter}`}>
-            {state.name} on Twitter
-          </a>
+          <a href={`https://twitter.com/${state.twitter}`}>Official Twitter</a>
```

Comment:
To help with screen reader users and prevent duplicate links, let's rewrite these as:

```
<span className="a11y-only">{state.name}'s</span>Official Twitter
```

Reply:
Good idea, I'll add that.